dashboard-elements/note.mdx

```diff
@@ -0,0 +1,23 @@
+---
+title: 'Note'
+description: 'This section explains how to create note dashboard elements and add them to your dashboard.'
+sidebarTitle: Note
+tags:
+    ['axiom documentation', 'documentation', 'axiom', 'dashboards', 'collaboration', 'queries', 'visualizations', 'filters', 'charts', 'simple query builder', 'advanced query builder']
+---
+
+import Prerequisites from "/snippets/dashboard-minimal-prerequisites.mdx"
+
+The note dashboard element adds a textbox to your dashboard that you can customise to your needs.
```

Comment:
```suggestion
The note dashboard element adds a textbox to your dashboard that you can customise to your needs. This can be helpful to give other members of your organization context about other dashboard elements.
```